### PR TITLE
Create BPK Summary Tabs_2014-18.sas

### PR DESCRIPTION
**Pull request by ocarena**
This is an update of Bridge Park data, including ACS 2014-2018, 2018 crime data, HMDA, DCRA, and births. @ptatian it is ready for your review.

### CODE REVIEW

**Comment by ptatian** [comment]
@ocarena You did! I will review your code.

File: Prog/BPK Summary Tabs_2014-18.sas [hunk 1]
```diff
@@ -0,0 +1,1493 @@
...
+			PctMrtgorigBlk_2011 = nummrtgorigblack_2011/nummrtgorigwithrace_2011;
+			PctMrtgorigBlk_2012 = nummrtgorigblack_2012/nummrtgorigwithrace_2012;
+			PctMrtgorigBlk_2013 = nummrtgorigblack_2013/nummrtgorigwithrace_2013;
+			PctMrtgorigBlk_2014 = nummrtgorigblack_2014/nummrtgorigwithrace_2014;
+			PctMrtgorigBlk_2015 = nummrtgorigblack_2015/nummrtgorigwithrace_2015;
+			PctMrtgorigBlk_2016 = nummrtgorigblack_2016/nummrtgorigwithrace_2016;
+			PctMrtgorigBlk_2017 = nummrtgorigblack_2017/nummrtgorigwithrace_2017;
+			PctMrtgorigBlk_2018 = nummrtgorigblack_2018/nummrtgorigwithrace_2018;
+				
+			PctMrtgorigAsn_2009 = nummrtgorigasian_2009/nummrtgorigwithrace_2009;
```
Comment (ptatian) [inline Prog/BPK Summary Tabs_2014-18.sas]: Should be `nummrtgorigasianpi_2009`, etc.

File: Prog/BPK Summary Tabs_2014-18.sas [hunk 2]
```diff
@@ -0,0 +1,1493 @@
...
+				
+			/*Births*/
+			
+			PctBirthsPrenatInad_2003 = births_prenat_inad_2003/births_w_prenat_2003; 
+			PctBirthsPrenatInad_2004 = births_prenat_inad_2004/births_w_prenat_2004; 
+			PctBirthsPrenatInad_2005 = births_prenat_inad_2005/births_w_prenat_2005; 
+			PctBirthsPrenatInad_2006 = births_prenat_inad_2006/births_w_prenat_2006; 
+			PctBirthsPrenatInad_2007 = births_prenat_inad_2007/births_w_prenat_2007; 
+			PctBirthsPrenatInad_2008 = births_prenat_inad_2008/births_w_prenat_2008; 
+			PctBirthsPrenatInad_2009 = births_prenat_inad_2009/births_w_prenat_2009;
```
Comment (ptatian) [inline Prog/BPK Summary Tabs_2014-18.sas:622]: Just noting there is a gap in the birth data in 2009. You will see missing data for this year.

File: Prog/BPK Summary Tabs_2014-18.sas [hunk 3]
```diff
@@ -0,0 +1,1493 @@
...
+			PctBirthsPrenatIntrAsn_2015 = births_prenat_intr_asn_2015/births_w_prenat_asn_2015;
+			PctBirthsPrenatIntrAsn_2016 = births_prenat_intr_asn_2016/births_w_prenat_asn_2016;
+
+			PctBirthsPrenatIntrBlk_2003 = births_prenat_intr_blk_2003/births_w_prenat_blk_2003;
+			PctBirthsPrenatIntrBlk_2004 = births_prenat_intr_blk_2004/births_w_prenat_blk_2004;
+			PctBirthsPrenatIntrBlk_2005 = births_prenat_intr_blk_2005/births_w_prenat_blk_2005;
+			PctBirthsPrenatIntrBlk_2006 = births_prenat_intr_blk_2006/births_w_prenat_blk_2006;
+			PctBirthsPrenatIntrBlk_2007 = births_prenat_intr_blk_2007/births_w_prenat_blk_2007;
+			PctBirthsPrenatIntrBlk_2008 = births_prenat_intr_blk_2008/births_w_prenat_blk_2008;
+			PctBirthsPrenatIntrBlk_2009 = births_prenat_intr_blk_2009/births_w_
```
Comment (ptatian) [inline Prog/BPK Summary Tabs_2014-18.sas:1274]: We should inflation adjust these income amounts to 2018 dollars for the table output. You can use the %dollar_convert() macro to do this.

```
%dollar_convert( medianmrtginc1_4m_2009, r_medianmrtginc1_4m_2009, 2009, 2018 )
%dollar_convert( medianmrtginc1_4m_2010, r_medianmrtginc1_4m_2010, 2010, 2018 )
etc.
```

**Review by ptatian — CHANGES_REQUESTED** [review]
Hi @ocarena. This looks good! Just a few small changes and then you should be done. Please let me know if you have questions. cc: @mcohenui

**Comment by mcohenui** [comment]
@ocarena good to merge

**Pull request merged**